src/casp/data_manager/README.rst

```diff
@@ -1,6 +1,9 @@
 Data Manager
 ============
 
+..
+    Are we still using bigquery?  Will we still be using it at the time when we start letting people actually use this thing?
+
```

Comment:
Good call. I woudl just mention it's posgres and remove the dialect link section

Reply:
This particular files talks about a lot of stuff that I think is BigQuery specific.  At least some of it is.  How much changed from the switch to postgres?  Like, are we still using `.mako` files for those queries?  I'm unclear on how much of the stuff in this file is actually accurate to how we're storing and querying cell data now.  @fedorgrab, do you have any input on this?

Reply:
Well this is still in use, but mako template files are all related to data ops scripts. And if we are about to change the repo structure, maybe this will go away. I think in the nearest future, we need to factor out all the ml / data ops code into a separate place which will make a DataManager more like a communicator with the postgres in cas. But now it is still used by the scripts, which use the SQL templates. I think it should be a part of a bigger conversation, maybe let's reserve some time for our next 1 hour meeting.

Reply:
As we discussed in the meeting, we're still using that in bq_ops scripts, so this should exists in the docs until we refactor the scripts and data manager.